test_data/std/keccakf16_memory_test.asm

```diff
@@ -0,0 +1,117 @@
+use std::machines::hash::keccakf16_memory::Keccakf16Memory;
+use std::machines::small_field::memory::Memory;
+use std::machines::small_field::add_sub::AddSub;
+use std::machines::range::Byte2;
+use std::machines::range::Bit12;
+
```

Comment:
Can you also call this test in riscv tests? Otherwise CI won't run it

Reply:
Added. Thanks for pointing out.